Dockerfile

```diff
@@ -9,38 +9,69 @@ RUN apt-get update \
     python-rosinstall \
     sudo \
     wget \
+    ros-eloquent-gazebo-dev \
+    ros-eloquent-gazebo-msgs \
+    ros-eloquent-gazebo-ros \
+    ros-eloquent-gazebo-plugins \
+    ros-eloquent-gazebo-ros-pkgs \
+    ros-eloquent-xacro \
+    ros-eloquent-desktop \
  && apt-get clean
 
-RUN rosdep update
+# Make sure everything is up to date before building from source
+RUN apt-get update \
+  && apt-get dist-upgrade -y \
+  && apt-get clean
+
+RUN /bin/sh -c 'echo ". /opt/ros/eloquent/setup.bash" >> ~/.bashrc' \
+  && /bin/sh -c 'echo ". /usr/share/gazebo/setup.sh" >> ~/.bashrc'
 
-RUN /bin/sh -c 'echo ". /opt/ros/melodic/setup.bash" >> ~/.bashrc' \
- && /bin/sh -c 'echo ". /usr/share/gazebo/setup.sh" >> ~/.bashrc'
+RUN mkdir  /workspace/drone_demo_ros2/src -p
+WORKDIR /workspace/drone_demo_ros2/src
+RUN git clone https://github.com/ahcorde/drone_demo.git -b ahcorde/ros2
+RUN git clone https://github.com/ahcorde/sitl_gazebo.git -b ahcorde/ros2 --recursive
+RUN git clone https://github.com/osrf/uav_testing.git -b ahcorde/ros2
+RUN git clone https://github.com/osrf/drone_demo.git -b rviz_aerial_plugins rviz_aerial_plugins
```

Comment:
This should be updated to the new standalone repo.

Reply:
In order to change this repo `https://github.com/ahcorde/drone_demo.git -b ahcorde/ros2` we need to work into merge this https://github.com/osrf/drone_demo/pull/18

The others two I changed it https://github.com/osrf/drone_demo/pull/23/commits/0174c0b1eec8d1119a517a24b5821061e5af5adc